### ISSUE TITLE
Problem with Importing Triangles Clause

### ISSUE THREAD

**Issue by emthornber** [pycollada/pycollada#76]
A triangles section with a count of zero but with white-space in the `<p>...</p>` section causes an exception when doing a FreeCAD import.

**XML**

				<triangles count="0" material="mat_symb_02B5E3C4_119">
					<input semantic="VERTEX" source="#_NC_mesh02B5E3C4-Vtx" offset="0" />
					<input semantic="TEXCOORD" source="#_NC_mesh02B5E3C4-TexCoord0" offset="1" set="0" />
					<input semantic="NORMAL" source="#_NC_mesh02B5E3C4-Normal" offset="2" />
					<p>					</p>

**Error**

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Applications/FreeCAD 2.app/Contents/Mod/Arch/importDAE.py", line 98, in insert
    read(filename)
  File "/Applications/FreeCAD 2.app/Contents/Mod/Arch/importDAE.py", line 115, in read
    col = collada.Collada(filename, ignore=[collada.DaeUnsupportedError])
  File "/Applications/FreeCAD 2.app/Contents/lib/python2.7/site-packages/collada/`__init__`.py", line 211, in __init__
    self._loadGeometry()
  File "/Applications/FreeCAD 2.app/Contents/lib/python2.7/site-packages/collada/`__init__`.py", line 290, in _loadGeometry
    G = geometry.Geometry.load(self, {}, geomnode)
  File "/Applications/FreeCAD 2.app/Contents/lib/python2.7/site-packages/collada/geometry.py", line 215, in load
    _primitives.append( triangleset.TriangleSet.load( collada, sourcebyid, subnode ) )
  File "/Applications/FreeCAD 2.app/Contents/lib/python2.7/site-packages/collada/triangleset.py", line 227, in load
    triset = TriangleSet(source_array, node.get('material'), index, node)
  File "/Applications/FreeCAD 2.app/Contents/lib/python2.7/site-packages/collada/triangleset.py", line 97, in `__init__`
    self.index.shape = (-1, 3, self.nindices)
<type 'exceptions.ValueError'>: cannot reshape array of size 1 into shape (3,3)
```

If the **p** section is truly empty (`<p></p>`) then the DEA file is imported successfully.

**Comment by jterrace** [comment]
Looks like `numpy.fromstring` returns a single element of zero when there are no values:

```
>>> numpy.fromstring('0    1 \t   \n2', dtype=numpy.int32, sep=' ')
array([0, 1, 2], dtype=int32)
>>> numpy.fromstring('', dtype=numpy.int32, sep=' ')
array([0], dtype=int32)
>>> numpy.fromstring('    \t     \n', dtype=numpy.int32, sep=' ')
array([0], dtype=int32)
```

That's... weird.

**Comment by jterrace** [comment]
Fixed with #78
